src/ifp/src/InitFloorplan.cc

```diff
@@ -227,16 +227,19 @@ void InitFloorplan::updateVoltageDomain(dbSite* site,
   // checks if a group is defined as a voltage domain, if so it creates a region
   for (dbGroup* group : block_->getGroups()) {
     if (group->getType() == dbGroupType::VOLTAGE_DOMAIN) {
-      dbRegion* domain_region = dbRegion::create(block_, group->getName());
-
+      dbRegion* domain_region = group->getRegion();
```

Comment:
Ok for now as we only create single boxes but this will need to deal with rectilinear regions eventually.